src/components/discovery/VariantSearchHeader.js

```diff
@@ -17,18 +21,6 @@ const VariantSearchHeader = ({dataType, addVariantSearchValues}) => {
     const handleAssemblyIdChange = (value) => {
 
         addVariantSearchValues({assemblyId: value});
-
-    // temp workaround for bug in Bento back end:
-    // files ingested with GRCh37 reference need to be searched using assemblyId "Other"
-
-    // so if assembly is "Other", pass that value to the form, but use
-    // "GRCh37" as the reference for gene lookup in Gohan
-        if (value === "Other") {
-            setLookupAssemblyId("GRCh37");
-            return;
-        }
-
-        setLookupAssemblyId(value);
```

Comment:
setLookupAssemblyId(value) needs to stay or variant search will break. 

only  the` if (value === "Other")` block needs to go.

Reply:
fixed